docs/src/main/paradox/index.md

```diff
@@ -16,6 +16,7 @@
 * [Running](running.md)
 * [Actor](actor.md)
 * [Flow](flow.md)
+* [Error](error.md)
```

Comment:
```suggestion
* [Error handling](error.md)
```

Reply:
that text is not used, it will take the `# ` heading of the page

Reply:
Oh, ok. I see.